backends-clickhouse/src/main/scala/io/glutenproject/execution/GlutenMergeTreePartition.scala

```diff
@@ -35,7 +35,8 @@ case class GlutenMergeTreePartition(
     engine: String,
     database: String,
     table: String,
-    tablePath: String,
+    relativeTablePath: String,
+    absoluteTablePath: String,
```

Comment:
why it needs this absolute table path ?  it seems that it does not use in CH backend.

Reply:
it's finally used in io.glutenproject.substrait.rel.ExtensionTableNode#getAbsolutePath